ddtrace/contrib/asgi/middleware.py

```diff
@@ -23,14 +27,14 @@ def bytes_to_str(str_or_bytes):
     return str_or_bytes.decode() if isinstance(str_or_bytes, bytes) else str_or_bytes
 
 
-def _extract_tags_from_scope(scope):
+def _extract_tags_from_scope(scope, config):
```

Comment:
We should probably change the global import `config` so that these variables don't shadow that definition.